src/transports/ws/bws.c

```diff
@@ -314,6 +344,71 @@ static void nn_bws_handler (struct nn_fsm *self, int src, int type,
         }
 
 /******************************************************************************/
+/*  CLOSING_USOCK state.                                                      */
```

Comment:
Just a little note.  I hate the use of enclosing asterisks in block comments.  It becomes a pain to maintain them, and it looks horrible if ever used with a non-monospaced font (I know don't do that!) I used to use the same comment style when I was young, but in my years since I've learned that they add little value -- the ones on the right side are especially annoying to have to maintain.  Usually just a single multiline comment with left aligned asterisks is sufficient to catch attention -- especially if surrounded by whitespace.